a10_octavia/controller/worker/tasks/service_group_tasks.py

```diff
@@ -69,20 +76,23 @@ class PoolDelete(task.Task):
     @axapi_client_decorator
     def execute(self, pool, vthunder):
         try:
+            raise ConnectionError
```

Comment:
I don't think this was meant to be left in

Reply:
yes you are right. My bad. Will remove this.